test/Sign.Core.Test/SignatureProviders/AggregatingSignatureProviderTests.cs

```diff
@@ -10,7 +10,7 @@ namespace Sign.Core.Test
 {
     public partial class AggregatingSignatureProviderTests
     {
-        private static readonly SignOptions _options = new(HashAlgorithmName.SHA256);
+        private static readonly SignOptions _options = new(HashAlgorithmName.SHA256, new Uri("http://timestamp.test"));
```

Comment:
Are there tests for timestamping signing assertion if timestamping server is provided?
If I understand correctly if it's timestamped, then we can verify it against same server, right?

Reply:
I'm not sure I understand the question, so I'll ask/answer a few questions I think you might be asking.

Q:  Are there automated tests that verify that a (syntactically) valid value was passed for the `--timestamp-url` option?
A:  Yes, [here](https://github.com/dotnet/sign/pull/540/files#diff-1a2ec79601f9b01f7dc6a290224d0c978bc8691aa175c4063d19b1dd0336bd1dR24-R32).

Q:  Are there automated tests that timestamping actually works end-to-end?
A:  No, that level of end-to-end testing isn't automated.

Q:  Does verifying a timestamp access the timestamping server?
A:  Usually, no.  A timestamping server provides a timestamp, and the timestamp contains the signing certificate.  Verifying the timestamp may involve chain building the certificate, to include revocation checking, but neither of those operations typically hit the timestamp server.

If you meant something else, let me know.